plugin/teksi_wastewater/interlis/interlis_model_mapping/interlis_importer_to_intermediate_schema.py

```diff
@@ -1821,8 +1821,8 @@ def _import_messresultat(self):
                 ),
                 measuring_duration=row.messdauer,
                 remark=row.bemerkung,
-                time=row.zeit,
-                value=row.wert,
+                time_point=row.zeit,
```

Comment:
@cymed Can you add a comment on this - else we will not remember, why this was adapted: e.g. rename from time to time_point and from value to measurement_value as xxx implemented time and value as SQL key words - prevent future conflict on PostgresSQL